pkgs/build-support/kernel/make-initrd-ng/src/main.rs

```diff
@@ -51,16 +50,15 @@ fn patch_elf<S: AsRef<OsStr>, P: AsRef<OsStr>>(mode: S, path: P) -> Result<Strin
     }
 }
 
-fn copy_file<P: AsRef<Path> + AsRef<OsStr>, S: AsRef<Path>>(
+fn copy_file<P: AsRef<Path> + AsRef<OsStr>, S: AsRef<Path> + AsRef<OsStr>>(
     source: P,
     target: S,
     queue: &mut NonRepeatingQueue<Box<Path>>,
 ) -> Result<(), Error> {
-    fs::copy(&source, target)?;
+    fs::copy(&source, &target)?;
 
     if !Command::new("ldd").arg(&source).output()?.status.success() {
-        //stdout(Stdio::inherit()).stderr(Stdio::inherit()).
-        println!("{:?} is not dynamically linked. Not recursing.", OsStr::new(&source));
```

Comment:
While we're at it, we could silence the `patchelf: cannot find section '.interp'. The input file is most likely statically linked` messages for the `--print-interpreter` call.

Reply:
Already done, I removed the stderr redirection ;)

Reply:
Ah, I see. I think the stderr redirection is useful for *all the other cases*, since it won't usually output anything unless something is genuinely wrong. But that's ok as is.